go.mod

```diff
@@ -2,7 +2,7 @@ module sigs.k8s.io/cluster-api-provider-vsphere
 
 go 1.23.0
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.9.0-rc.0.0.20250317123349-9e7afa749358
+replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.10.0-beta.0.0.20250319172205-4f022adb6164
```

Comment:
wondering why this is this instead of `v1.10.0-beta.0`?

But ok for me

Reply:
Unfortunately can't because https://github.com/kubernetes-sigs/cluster-api/pull/11991 didn't make it into the beta.0